mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.kt

```diff
@@ -990,6 +1004,25 @@ class MainActivity : AbstractBaseActivity(), ConnectionFactory.UpdateListener {
         }
     }
 
+    private fun pinHabpanelShortcut(context: Context) = GlobalScope.launch {
+        val intent = Intent(context, MainActivity::class.java)
+            .setAction(ACTION_HABPANEL_SELECTED)
+        val shortcutInfo = ShortcutInfoCompat.Builder(context, "habpanel-" + System.currentTimeMillis())
```

Comment:
Is this really required to be made unique by adding the timestamp?

Reply:
I've to investigate this. On one of my devices I cannot pin this shortcut more than once, even with this timestamp. Maybe launchers check for the same action, name, icon or a combination of this.

Reply:
No, the timestamp isn't needed.